### PR TITLE
Grammatical adjustments of `page models` usage guide to resolve issue #9398

### PR DESCRIPTION
**Pull request by activus-d**
Corrections to the grammar in [Pages model](https://github.com/wagtail/wagtail/blob/main/docs/topics/pages.md#page-models) section of the Usage guide, which were not addresses in [issue #9395](https://github.com/wagtail/wagtail/pull/9395). This will enhance its readability and resolve issue #9398.

### CODE REVIEW

**Comment by rahul3002** [comment]
I would like to work on this issue

**Comment by lb-** [comment]
> I would like to work on this issue

@rahul3002 This is not an issue but rather a PR (pull request), so no need for additional work on this. 

It's just waiting on a review from the core team.

File: docs/topics/pages.md
Comment (thibaudcolas) [inline docs/topics/pages.md:3]: I find the existing content much clearer.

```suggestion
Each page type (a.k.a. content type) in Wagtail is represented by a Django model. All page models must inherit from the {class}`wagtail.models.Page` class.
```

File: docs/topics/pages.md
Comment (thibaudcolas) [inline docs/topics/pages.md:170]:
```suggestion
With every Wagtail Page you are able to add a helpful description text, similar to a `help_text` model attribute. By adding `page_description` to your Page model you'll be adding a short description that can be seen when you create a new page, edit an existing page or when you're prompted to select a child page type.
```

File: docs/topics/pages.md
Comment (thibaudcolas) [inline docs/topics/pages.md:216]:
```suggestion
To retrieve the full URL (including the protocol and domain), use `Page.get_full_url(request)`. Whenever possible, the optional `request` argument should be included to enable per-request caching of site-level URL information.
```

**Review by thibaudcolas — APPROVED** [review]
Thank you @activus-d! I’ve reverted some of the proposed changes as I don’t think they clarify things, but there’s plenty of good changes in there.

**Pull request merged**
